src/UI/Areas/Identity/Pages/Account/ResetPassword.cshtml.cs

```diff
@@ -31,12 +31,13 @@ public class InputModel
             [Compare("Password", ErrorMessage = "The password and confirmation password do not match.")]
             public string ConfirmPassword { get; set; }
 
+            [Required]
             public string Code { get; set; }
 
-            public virtual IActionResult OnGet(string code = null) => throw new NotImplementedException();
-
-            public virtual Task<IActionResult> OnPostAsync() => throw new NotImplementedException();
         }
+        public virtual IActionResult OnGet(string code = null) => throw new NotImplementedException();
```

Comment:
Is there any way to detect this defect via some kind of reflection test?  Is it safe to say that every page in the default UI should have these two as public virtuals?

Reply:
I'm trying to find a way of detecting this at run time, but I haven't come up with something that is not "flaky". I'll chat with Ryan about it.